packages/commonwealth/client/scripts/views/components/component_kit/new_designs/CWSearchBar.tsx

```diff
@@ -0,0 +1,148 @@
+import React, { FC, ChangeEvent, useState, useEffect } from 'react';
+import { MagnifyingGlass } from '@phosphor-icons/react';
+import useAutocomplete from '@mui/base/useAutocomplete';
+
+import { ComponentType } from '../types';
+import { CWTag } from './CWTag';
+
+import 'components/component_kit/new_designs/CWSearchBar.scss';
+import { IconName } from '../cw_icons/cw_icon_lookup';
+import { getClasses } from '../helpers';
+import { ValidationStatus } from '../cw_validation_text';
+
+type BaseSearchBarProps = {
+  autoComplete?: string;
+  autoFocus?: boolean;
+  containerClassName?: string;
+  defaultValue?: string | number;
+  value?: string | number;
+  iconLeft?: IconName;
+  iconLeftonClick?: () => void;
+  inputValidationFn?: (value: string) => [ValidationStatus, string] | [];
+  label?: string | React.ReactNode;
+  maxLength?: number;
+  name?: string;
+  onInput?: (e) => void;
+  onenterkey?: (e) => void;
+  onClick?: (e) => void;
+  placeholder?: string;
+  tabIndex?: number;
+  manualStatusMessage?: string;
+  manualValidationStatus?: ValidationStatus;
+};
+
+type InputStyleProps = {
+  inputClassName?: string;
+  disabled?: boolean;
+};
+
+type InputInternalStyleProps = {
+  hasLeftIcon?: boolean;
+  isTyping?: boolean;
+};
+
+type SearchBarProps = BaseSearchBarProps &
+  InputStyleProps &
+  InputInternalStyleProps &
+  React.HTMLAttributes<HTMLDivElement>;
+
+const communities = [
+  'Altitude',
+  'Terra Classic',
+  'Osmosis',
+  'Qwoyn Network',
+  '1inch',
+  'Stargate Finance',
+  'Timeless',
+  'Terra Agora',
+  'Injective',
+  'Juno',
+  'Common',
+];
```

Comment:
this variable cannot be defined here. see the comment below

Reply:
Removed.